lambda/custom/document/renderPage.json

```diff
@@ -0,0 +1,138 @@
+{
+    "type": "APL",
+    "version": "1.0",
+    "settings": {},
+    "theme": "dark",
+    "import": [
+            { 
+                "name" : "alexa-styles", 
+                "version" : "1.0.0" 
+            } 
+        ],
+    "resources": [
+        { 
+            "colors" : {
+               "SkillBackgroundColor" : "blue" , 
+               "SkillHighlightTextColor"  : "yellow" , 
+               "SkillTextColor" : "white"
+            }
+        } 
+        
+        ],
+    "styles": {
+        
+        
+    },
```

Comment:
Please clean up all of the extra whitespace and spacing in this document.

Reply:
will do